release-notes/con-relnotes-known-issues.adoc

```diff
@@ -3,6 +3,14 @@
 
 This section lists known issues with {product} {product-version}:
 
+Catalog-import function can leak GitLab access tokens on the frontend when base64-encoded GitLab token includes a newline at the end of the string link:https://issues.redhat.com/browse/RHIDP-1734[[RHIDP-1734]]::
```

Comment:
I assume we mean the RHDH frontend? Maybe specify.

Reply:
yeah the frontend plugin which is on the front end of RHDH, I assume. See suggested fix https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/138#discussion_r1552092225